src/transactions/reducer.ts

```diff
@@ -6,6 +6,7 @@ import { RootState } from 'src/redux/reducers'
 import { Actions, ActionTypes } from 'src/transactions/actions'
```

Comment:
can we also make the necessary changes to persist standby transactions across app sessions? https://github.com/valora-inc/wallet/blob/645f51785d23acd16e6d8e45ef09b2595d68d978/src/transactions/reducer.ts#L63-L70